package.json

```diff
@@ -5,10 +5,10 @@
   "type": "module",
   "sideEffects": false,
   "source": "./src/index.ts",
-  "types": "./dist/src/index.d.ts",
   "main": "./dist/three-to-cannon.cjs",
   "module": "./dist/three-to-cannon.esm.js",
   "exports": {
+    "types": "./dist/src/index.d.ts",
```

Comment:
Thanks @azad-source! Do you mind keeping the old declaration as well, so we have both `types` and `exports.types`?

Reply:
Sure, that's fine idea to keeping both the old declaration and the new one. This is actually the better approach to ensure compatibility with older Node.js versions. 

Thank you for creating such a useful library!